spec/example_app/db/schema.rb

```diff
@@ -36,10 +36,9 @@
     t.string   "queue"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.index ["priority", "run_at"], name: "delayed_jobs_priority", using: :btree
```

Comment:
It looks like Rails is switch `add_index` => `t.index`. I would keep it. Its new for Rails 5 and besides, its going to have the same effect as everything else. Also, its a smaller schema.rb :p